index.js

```diff
@@ -155,14 +155,24 @@ Notifications.localNotification = function(details: Object) {
  */
 Notifications.localNotificationSchedule = function(details: Object) {
 	if ( Platform.OS === 'ios' ) {
-		this.handler.scheduleLocalNotification({
+		const iosDetails = {
 			fireDate: details.date.toISOString(),
 			alertBody: details.message,
-			userInfo: details.userInfo
-		});
+			userInfo: details.userInfo,
+			repeatInterval: details.repeatType
+		}
+		// ignore Android only repeatType
+		if (!details.repeatType || details.repeatType === 'time') {
+			delete iosDetails.repeatInterval;
```

Comment:
I think this case is already handled in RTCPushNotificationManager.m with the RCTConvert... 😅 I don't do obj-C so I can't explain how it work. But if you try to put a value not matching the RCT enum, it just do not set a repeat interval...so no need to delete it...